tools/platyPS/ValidateHelp.ps1

```diff
@@ -0,0 +1,179 @@
+                        }
+                        else
+                        {
+                            break
+                        }
+                    }
+
+                    if (!($foundSynopsis))
+                    {
+                        $fileErrors += "`tNo synopsis found"
+                    }
+                }
+                "## DESCRIPTION"
+                {
+                    $foundDescription = $False
+                    $idx++
+
+                    # Check each line between DESCRIPTION and EXAMPLES for any text
+                    for (;;)
+                    {
+                        $foundDescription = $foundDescription -or (!([string]::IsNullOrWhiteSpace("$($content[$idx])")))
+
+                        if ($content[$idx+1] -notcontains "## EXAMPLES")
+                        {
+                            $idx++
+                        }
+                        else
+                        {
+                            break
+                        }
+                    }
+
+                    if (!($foundDescription))
+                    {
+                        $fileErrors += "`tNo description found"
+                    }
+                }
+                "## EXAMPLES"
+                {
+                    # Move the index to the start of the PowerShell code
+                    while ($content[$idx] -notcontains "``````")
+                    {
+                        $idx++
+                    }
+
+                    # Check for the platyPS example template
+                    # 
+                    # ```
+                    # PS C:\> {{ Add example code here }}
+                    # ```
+                    # 
+                    if ($content[$idx+1] -contains "PS C:\> {{ Add example code here }}")
+                    {
+                        $fileErrors += "`tNo examples found"
+                    }
+
+                    # Check for other missing example formats (such as empty)
+                    # 
+                    # ```
+                    # 
+                    # ```
+                    # 
+                    if ([string]::IsNullOrWhiteSpace("$($content[$idx+1])"))
+                    {
+                        $fileErrors += "`tNo examples found"
+                    }
+                }
+                "@{Text=}"
+                {
+                    # This case occurs when there is no description provided for a parameter
+                    $parameter = $content[$idx-1].Substring(5)
+                    $fileErrors += "`tNo description found for parameter $parameter"
+                }
+                "## OUTPUTS"
+                {
+                    $foundOutput = $False
+                    $idx++
+
+                    # Check each line between OUTPUTS and NOTES for any text
+                    for (;;)
+                    {
+                        $foundOutput = $foundOutput -or (!([string]::IsNullOrWhiteSpace("$($content[$idx])")))
```

Comment:
That was something I thought about but didn't continue with. I will find a way to separate the output warnings from the regular errors in the script.